README.md

```diff
@@ -15,13 +14,11 @@
 
 [Live Demo Link](https://livedemo.com)
 
-
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
```

Comment:
- [ ] Kindly provide the necessary information and commands to run (`npm install`, `npm start`) that a user would need to set up and run your project locally on their machine.
- [ ] Also, it would be good to remove all the empty sections in your README file and also remove the live demo link as it is not currently working. This will make your repo more professional 👍

Reply:
All good now 👍🏽